curvefs/src/client/main.c

```diff
@@ -49,6 +49,7 @@ static const struct fuse_lowlevel_ops curve_ll_oper = {
     .release    = FuseOpRelease,
```

Comment:
If you remove the FuseOpRelease impliment,  ”.release = FuseOpRelease“ also should remove.